.storybook/webpack.config.js

```diff
@@ -44,20 +58,21 @@ module.exports = (storybookBaseConfig, configType) => {
           },
         },
       ],
+      include: path.resolve(__dirname, '../'),
     },
     {
       test: /\.svg$/,
       use: [
         {
-          loader: '@svgr/webpack',
+          loader: 'react-svg-loader',
```

Comment:
What was the reasoning for this change?

Reply:
good point. I think I was following along the progress made in the other PR. Looks like it's not really necessary as I see it is working without this change. Should I go ahead and remove `react-svg-loader`?